tests/mgbench/compare_results.py

```diff
@@ -79,45 +79,45 @@ def recursive_get(data, *args, value=None):
 
 def compare_results(results_from, results_to, fields):
     ret = {}
+
+    with open("compare_ignore.txt", "r") as f:
```

Comment:
@brunos252 I would pass this as a runtime flag + by default nothing should be ignored 😄